src/components/Content/contactUs/ContactUsContent.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useRouter } from 'next/router';
+
+type ContactUsProps = {
+  children: React.ReactNode;
+};
+const ContactUs: React.FC<ContactUsProps> = ({ children }) => {
```

Comment:
Don't create ContactUsProps and implement here. 

Instead use "React.PropsWithChildren<{}>"